src/libcudart.jl

```diff
@@ -0,0 +1,97 @@
+module CUDArt_gen  # the generated module
+
+# Determine the location of the CUDArt library.
+function find_cudart_lib()
+    return @windows? begin
+        # location of cudart64_xx.dll or cudart32_xx.dll have to be in PATH env var
+        # ex: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v6.5\bin
+        # (by default, it is done by CUDA toolkit installer)
+
+        const dllnames = ["cudart$(WORD_SIZE)_$ver" for ver in (75, 70, 65, 60, 55, 50, "50_35")]
+        Libdl.find_library(dllnames, [""])
+    end : begin # linux or mac
+        const libdir = (WORD_SIZE==64) ? "lib64" : "lib"
+        const locations = ["/usr/local/$libdir", "/opt/$libdir"]
+        const prefixes = ["cuda", ["cuda-$ver" for ver in ("7.5", "7.0", "6.5", "6.0", "5.5", "5.0")]...]
+        const paths = [joinpath(loc, pref) for loc in locations, pref in prefixes]
+        Libdl.find_library(["libcudart", "cudart"], vec(paths))
+    end
+end
+
+const libcudart = find_cudart_lib()
+
+if isempty(libcudart)
+    error("CUDA runtime API library cannot be found")
+end
+
+# Determine the version of the CUDArt library.
+function cudart_version()
+    a = Cint[0]
+    ccall((:cudaRuntimeGetVersion,libcudart),UInt32,(Ptr{Cint},),a)
```

Comment:
I'm getting this same behaviour on my laptop when using the on-board GPU. Does this happen to be the case for you? It's because in that case, `cudaRuntimeGetVersion` gives a bogous return value, `0x00000026` in my case.

How did you get here? Maybe I need to add a better error message somewhere?